roles/matrix-synapse/defaults/main.yml

```diff
@@ -326,6 +336,16 @@ matrix_synapse_additional_loggers: []
 # You may wish to use this together with `matrix_synapse_container_additional_volumes` or `matrix_synapse_container_extra_arguments`.
 matrix_synapse_app_service_config_files: []
```

Comment:
```suggestion
# Also see `matrix_synapse_app_service_config_files_final`
matrix_synapse_app_service_config_files: []
```

(well, or whatever name you decide to go with when renaming the `matrix_synapse_config_files` varialbe, as proposed in another review comment)